### PR TITLE
[release/v2.25] add 2.25.10 changelog

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
2.25.10 will only ship #13737.

**What type of PR is this?**
/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
NONE
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: docs/changelogs/CHANGELOG-2.25.md [hunk 1]
```diff
@@ -10,6 +10,15 @@
...
 - [v2.25.9](#v2259)
+- [v2.25.10](#v22510)
+
+## v2.25.10
+
+**GitHub release: [v2.25.10](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.10)**
+
+### Bugfixes
+
+- Fix failure to migrate Cilium `ApplicationInstallations` to new `valuesBlock` field. ([#13736](https://github.com/kubermatic/kubermatic/pull/13736))
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.25.md]: I updated the PR description for the changelog to be:

> Fix failure to migrate Cilium `ApplicationInstallations` to new `valuesBlock` field (this has been an undocumented change in KKP 2.25.9)

Can we adjust please?

**Review by embik — CHANGES_REQUESTED** [review]

File: docs/changelogs/CHANGELOG-2.25.md (hunk 1 above)
Comment (embik) [reply docs/changelogs/CHANGELOG-2.25.md]: Pushed myself to move this along :)

Comment (xrstf) [reply docs/changelogs/CHANGELOG-2.25.md]: Well I cannot review my own PR. ;)

**Review by embik — APPROVED** [review]

**Pull request merged**